frontend/packages/console-app/src/components/user-preferences/language/const.ts

```diff
@@ -1,8 +1,8 @@
 export const supportedLocales = {
-  en: 'English',
-  zh: '中文',
-  ko: '한국어',
-  ja: '日本語',
+  'en-US': 'English',
```

Comment:
I believe this could change how numbers and times are formatted for English-speaking users in other countries. Something to test?

I think we just want to change `zh` to `zh-CN` and leave the others as-is.

Reply:
@spadgett @jhadvig Updated! PTAL.